pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -317,6 +317,45 @@ func (br *OVSBridge) GetDatapathID() (string, Error) {
 	}
 }
 
+func (br *OVSBridge) WaitForDatapathID(timeout time.Duration) (string, Error) {
```

Comment:
There is a `GetDatapathID()` method and a `GetOFPort(ifName string, waitUntilValid bool)` method. Does it make sense to add a `waitUntilValid` to the former to unify them and probably to reduce some code?

Reply:
I am not a big fan of the current `GetOFPort` method, but maybe I can come up with a solution that I like that avoid the code duplication

Reply:
Got it, I was just not sure if you missed it. No problem from me if it looks worse by unifying them.